python_modules/dagster/dagster/_core/instance/config.py

```diff
@@ -251,6 +251,7 @@ def sensors_daemon_config() -> Field:
         {
             "use_threads": Field(Bool, is_required=False, default_value=False),
             "num_workers": Field(int, is_required=False),
+            "num_enqueue_workers": Field(int, is_required=False),
```

Comment:
should have a description to discern the difference between these two workers, doubt we can capture via name 

nit: is there something better than enqueue? submit?